libsrc/leddevice/LedDeviceWs2801.cpp

```diff
@@ -4,6 +4,10 @@ LedDeviceWs2801::LedDeviceWs2801(const QJsonObject &deviceConfig)
 	: ProviderSpi()
 {
```

Comment:
when you set latschTime before init, then this value is taken as default when not configured